src/OpenTelemetry.Extensions.Enrichment/EnrichmentExtensions.cs

```diff
@@ -0,0 +1,91 @@
+// <copyright file="EnrichmentExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Linq;
+using Microsoft.Extensions.DependencyInjection;
+using OpenTelemetry.Internal;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Extensions.Enrichment;
```

Comment:
Consider separate file for extensions on TracerProviderBuilder and IServiceCollection and use MS.Ex.DI ns for the IServiceCollection extensions.

Reply:
Done